src/core/cheats.c

```diff
@@ -802,3 +803,30 @@ void mCheatDeviceDeinit(struct mCPUComponent* component) {
 		}
 	}
 }
+
+void mCheatToggle(struct mCheatDevice* device) {
+	size_t i;
+	bool cheatsDisabled = true;
+	for (i = 0; i < mCheatSetsSize(&device->cheats); ++i) {
```

Comment:
Initial loop checks to see if any cheats are enabled, and if so disables them and saves them.